action.yml

```diff
@@ -38,6 +38,11 @@ inputs:
       A boolean indicating whether the relase being created or updated is a prerelease.
     required: false
     default: ''
+  commitish:
```

Comment:
Missing under actions input in the readme: https://github.com/release-drafter/release-drafter#action-inputs

Reply:
Thanks for the review, @jetersen ! Resolved in 0836e2a